examples/validation-app/README.md

```diff
@@ -19,12 +19,10 @@ instance with the in-memory storage.
    @property({
     type: 'string',
     required: true,
-    // --- add jsonSchema -----
```

Comment:
you don't want to keep the comment? :) 
I was thinking that it might be clearer to users what has been added apart from the model generation bits. 
Perhaps:
```
// --- add jsonSchema for validation -----
```

Reply:
Re-added with some modification.